JavaAssignments/src/amarja/Arraycontnum.java

```diff
@@ -0,0 +1,34 @@
+package amarja;
+import java.util.*;
+//Question 3 Program to find array contains number 
+public class Arraycontnum {
+	boolean findnum(int arr[],int num)
+	{	boolean flag=false;
+		for (int index=0;index<arr.length;index++)
+		{
+			if (arr[index]==num)
+				flag=true;
+		}
+		return flag;
+	}
+	public static void main (String args[])
+	{
+		Scanner sc=new Scanner(System.in);
+		System.out.println("Enter size of array");
+		int size=sc.nextInt();
+		System.out.println("Enter values of array");
```

Comment:
Please create another function to take int value from user. 
Write minimum code in main method as main method logic wont be reusable by other class.

Reply:
I had created another function and was using it eveywhere but I was getting the connection error for input stream.Hence had to write it in main method
Exception in thread "main" java.lang.NullPointerException
	at amarja.Arraycontnum.main(Arraycontnum.java:22)


please sugest ..As per your suggestion on skyp ehad written in main method.still getting errir

Reply:
Tried again able to create a seperate function .Comments incorporate.Please review